cost-analyzer/templates/_helpers.tpl

```diff
@@ -1421,6 +1429,22 @@ Groups is only used when using simple RBAC.
     {{- end }}
 {{- end }}
 
+{{- define "authMasterKeyEnabled" -}}
+  {{- if or (.Values.saml).enabled (.Values.oidc).enabled -}}
+    {{- if or (.Values.saml).apiMasterKey (.Values.oidc).apiMasterKey -}}
+        {{- printf "true" -}}
+      {{- else -}}
+        {{- if or (.Values.saml).apiMasterKeySecret (.Values.oidc).apiMasterKeySecret -}}
+          {{- printf "true" -}}
+        {{- else -}}
+          {{- printf "false" -}}
+        {{- end -}}
+    {{- end -}}
+  {{- else -}}
+    {{- printf "false" -}}
+  {{- end -}}
+{{- end -}}
```

Comment:
Indentation here is a little wonky, but I think that's just a cosmetic issue.

Reply:
Yeah, likely just a weird copy/paste on my part. Fixed.